homeassistant/components/person/icons.json

```diff
@@ -11,5 +11,13 @@
     "reload": {
       "service": "mdi:reload"
     }
+  },
+  "triggers": {
+    "entered_home": {
+      "trigger": "mdi:account-arrow-left"
+    },
+    "left_home": {
+      "trigger": "mdi:account-arrow-right"
+    }
```

Comment:
I wish these icons were a bit clearer (specifically the "entered" case). Alternatives might be `mdi:home-import-outline` and `mdi:home-export-outline`, but I’m not sure those are actually better than what you already have here.